### ISSUE TITLE
No species found in homo

### ISSUE THREAD

**Issue by snacktavish** [phylotastic/datelife#86]
```
new_q<-datelife::make_datelife_query(input="homo", get_spp_from_taxon = TRUE)
```

**Comment by bomeara** [comment]
https://tree.opentreeoflife.org/curator/study/edit/pg_2656/?tab=home should be pulled in as it has homo

**Comment by LunaSare** [comment]
Oh, I think it is because they are all extinct or subspecies in the OpenTree synth tree.
That's the criterion used in the get_spp_from_taxon flag.
Should we add a flag that keep extinct? And one for subspecies?

**Comment by bomeara** [comment]
yes on extinct. Odd about subspecies -- I think part of the problem is OToL has no homo sapiens at all, just the subspecies. This feels like a flaw in OToL, though.
